apps/admin/backend/src/app.ts

```diff
@@ -550,9 +530,11 @@ function buildApi({
 
       return [
         {
-          image: ballotImage.normalized,
+          image: toDataUrl(await loadImageData(image), 'image/jpeg').slice(
+            'data:image/jpeg;base64,'.length
+          ),
```

Comment:
OK, two thoughts here I think:
- You're suggesting this could just be the full data URL. I think that's a good idea. I thought about changing it but ended up just leaving it.
- At a later point we could return a URL that points back to the server.  I don't think you're suggesting doing that here but yeah, definitely, we could. I experimented with that and it works, I just didn't see the advantage at the moment but if someone made a compelling case for file storage and served images, I'm in.